File: DataStructures/LinkedList/PrintLinkedList.cs
using System;

public class LinkedList {
    Node head;


    public static class  Node {
        public int data;
        public Node next;
        public Node(int d)
        {
            data = d;
            next = null;
        }
    }


public void printList()
{
    Node  n = head;
    while(n != null)
    {
        Console.Write(n.data + " ");
        n = n.next;
    }

}


public static void Main(String[] args)
{
    LinkedList llist = new LinkedList();
    llist.head = new Node(1);
    Node second = new Node(2);
    Node third = new Node(3);

    llist.head.next = second;
    second.next = third;
    third.next = null;

    llist.printList();
}

}

File: DataStructures/LinkedList/PrintLinkedList.js
<script>
var head; 

class Node {
    constructor(val) {
        this.data = val;
        this.next = null;
    }
}

function printList()
{
    var n = head;
    while(n != null){
        document.write(n.data + "");
        n= n.next;
    }
}

var head = new Node(1);
var second = new Node(2);
var third = new Node(3);

head.next = second;
second.next = third;

printList();

</script>
//scripts are optional depends if writing into node or browser
//best to leave the scripts 